test/cpp/client/client_channel_stress_test.cc

```diff
@@ -253,24 +273,29 @@ class ClientChannelStressTest {
       request.set_message("test");
       EchoResponse response;
       {
-        std::lock_guard<std::mutex> lock(stub_mutex_);
-        Status status = stub_->Echo(&context, request, &response);
+        std::lock_guard<std::mutex> lock(client_handle->stub_mutex);
```

Comment:
I wondered the same thing but didn't feel like messing with it.  But since you noticed the same thing, I'll try removing it and see if anything breaks.

Reply:
It looks like it works fine without this.